dotcom-rendering/src/web/components/ArticleContainer.tsx

```diff
@@ -79,6 +79,7 @@ const adStyles = css`
 			.ad-slot {
 				/* from tablet the ad slot will stretch to the full width of the container and the iframe will be centred by the text-align: center; on the container */
 				flex: 1;
+				align-self: flex-start;
```

Comment:
Can we add a comment please to explain the magic :)

Reply:
You've got it!